react/features/authentication/components/native/WaitForOwnerDialog.tsx

```diff
@@ -97,9 +106,11 @@ class WaitForOwnerDialog extends Component<IProps> {
  */
 function mapStateToProps(state: IReduxState) {
     const { membersOnly, lobbyWaitingForHost } = state['features/base/conference'];
+    const serverURL = getDefaultURL(state);
```

Comment:
We don't want the default one, we need to check the current one.

Reply:
I was a bit confused because getDefaultURL returns getServerURL that returns the current one. Will update.

Reply:
Yes, but you can join a meeting in a server which is neither the default nor the configured one. We need to extract it from the current location URL.

Reply:
Update.